src/mlpack/methods/ann/layer/elu_impl.hpp

```diff
@@ -49,15 +52,18 @@ void ELU<InputDataType, OutputDataType>::Forward(
     const InputType&& input, OutputType&& output)
 {
   Fn(input, output);
+
+  if (not deterministic)
```

Comment:
We don't use the `not` keyword anywhere else (actually I didn't even know until just now it was supported in C++), so maybe for consistency we should go with `!` here instead?